koku/masu/external/downloader/gcp/gcp_report_downloader.py

```diff
@@ -316,14 +326,22 @@ def get_manifest_context_for_date(self, date):
         """
         reports_list = []
         if self.storage_only and not self.ingress_reports:
-            LOG.info("Skipping ingest as source is storage_only and requires ingress reports")
+            log_msg = "Skipping ingest as source is storage_only and requires ingress reports"
+            LOG.info(log_json(self.tracing_id, log_msg, self.context))
             return reports_list
         dh = DateHelper()
         if isinstance(date, datetime.datetime):
             date = date.date()
+        log_base = (
+            "New manifest to be created:"
+            f" dates {self.scan_start} - {self.scan_end}"
+            f" ingress: {bool(self.ingress_reports)}"
+        )
         if self.ingress_reports:
             manifest = self._generate_monthly_pseudo_manifest(date)
+            log_msg = log_base + f" Manifest Data: {str(manifest)}"
```

Comment:
String concatenation has additional overhead (all strings are built before being concatenated) compared to a single f-string operation.

The `__str__` method is called before default on objects inside f-strings. The `repr` can also be used: `{manifest!r}`.
```suggestion
            log_msg = f"{log_base} Manifest Data: {manifest}"
```